region/maridia/inner-green/East Pants Room.json

```diff
@@ -194,7 +194,8 @@
             }}
           ]}
         ]}
-      ]
+      ],
+      "note": "It is recommended to shoot the ceiling block immediately upon room entry to bring one of the Puyos down to help once getting up there."
```

Comment:
```suggestion
      "note": "It is recommended to shoot the ceiling block immediately upon room entry to bring one of the Puyos down so that it will not be a problem later."
```